README.md

```diff
@@ -83,10 +83,17 @@ Envless is licensed under the GNU Affero General Public License v3.0 License - s
 <h2 id="contributors">✨ Contributors</h2>
 
 + We ❤️ contributors! Feel free to contribute to this project!. 
++ Any contributions you make are truly appreciated.
++ Let's continue contributing to keep the community active and growing.
 + We also welcome you to join our [Slack](https://dub.sh/envless-slack) or [Discord](https://dub.sh/envless-discord) community for either support or contributing guidance.
 
 <a href="https://github.com/envless/envless/graphs/contributors">
   <p>
     <img src="https://contrib.rocks/image?repo=envless/envless" alt="A table of avatars from the project's contributors" />
   </p>
 </a>
+<h2>🙏 Support</h2>
+<p>
+Don't forget to leave a star ⭐️
```

Comment:
Thanks for adding this, I would love to have a gif showing how to star, something like this (but full width) would be great 🌟 

Repo: https://github.com/calcom/cal.com#stay-up-to-date

Reply:
I will be thankful, If you let me Know, How Can I do this one.